lib/blog-feed-loader.js

```diff
@@ -1,2 +1,6 @@
-// TODO: Replace this with a real implementation.
-module.exports = require('../test/browser/stub-blog-feed-loader');
+if (process.env.NODE_ENV !== 'production' &&
```

Comment:
I understand why this check is needed but can't figure out how it is related to https://github.com/webpack/docs/wiki/list-of-plugins#defineplugin.  :no_mouth:  We don't define any Google API variable [here](https://github.com/toolness/teach.webmaker.org/blob/integrate-google-feed-api-for-reals-this-time-take-2/webpack.config.js#L42) anyway?